Fix IsFullScreenMode signature in chrome's fullscreen_ozone.cc

Chrome build fails when use_ozone is true.

R=sky@chromium.org, warx@chromium.org
BUG=496681

Review-Url: https://codereview.chromium.org/2562233002
Cr-Commit-Position: refs/heads/master@{#438096}
diff --git a/chrome/browser/fullscreen_ozone.cc b/chrome/browser/fullscreen_ozone.cc
index ad6b84e..489fd05 100644
--- a/chrome/browser/fullscreen_ozone.cc
+++ b/chrome/browser/fullscreen_ozone.cc
@@ -6,7 +6,7 @@
 
 #include "services/service_manager/runner/common/client_util.h"
 
-bool IsFullScreenMode() {
+bool IsFullScreenMode(int64_t display_id) {
   if (service_manager::ServiceManagerIsRemote()) {
     // TODO: http://crbug.com/640390.
     NOTIMPLEMENTED();

